pycbc/inference/distributions.py

```diff
@@ -192,6 +193,42 @@ def _bounded_from_config(cls, cp, section, variable_args,
     return cls(**dist_args)
 
 
+def get_kde_from_file(params_file, params=None):
+    """Reads the values of one or more parameters from an hdf file and
+    computes the kernel density estimate (kde).
+
+    Parameters
+    ----------
+    params_file : str
+        The hdf file that contains the values of the parameters.
+    params : {None, list}
+        If provided, will just use the values for the given parameter.
+        Otherwise, uses the values for each parameter in the file.
+    Returns
+    -------
+    values
+        Array with the values of the parameters.
+    kde
+        The kde from the parameters.
+    """
+    try:
+        f = h5py.File(params_file, 'r')
+    except:
+        raise ValueError('File not found.')
+    if params is not None:
+        if str(params)[0] != '[':
```

Comment:
If you're checking the type of ``params`` is a list. Why not use ``type``? Eg. a simple example:
```
>>> a = [0,1,2]
>>> type(a) == list
True
```